OpenTrons/opentrons-api
# fix(protocol-designer): fix issues in select pipette screen

# Overview
fix issues in select pipette screen

close AUTH-762 RQA-3265 RQA-3267 RQA-3269



## Test Plan and Hands on Testing
AUTH-762
1. select a pipette
2. click add a pipette button
3. select a pipette and tips
4. click go back
check that PD displays the screen before the step3 (like the following)

![2024-10-02 16_14_30-NVIDIA GeForce Overlay DT](https://github.com/user-attachments/assets/07e6c2a3-6733-4571-94c1-802d1c58e9f1)

RQA-3265 RQA-3269
1. select a pipette
check the info card has remove button and switch pipettes button is displayed
2. click remove button in the info card
check only one EmptySelectorButton is displayed and switch pipettes button is not displayed

RQA-3267
1. select a pipette
add a pipette and tips button is displayed
2. click swap pipettes
check add a pipette and tips button is below the pipette info
3. click add pipette and tips 
check add a pipette and tips button is not in the wizard



## Changelog



## Review requests



## Risk assessment
low
